### PR TITLE
Ready for review

### PR DESCRIPTION
**Pull request by BrittanyBlake**
## Positioning and Floating Elements: New York Times Clone
-Added a fixed navigation bar
-Added article content
-Added embedded images and videos
-Added aside bar
-Added a footer

[Live Version](https://brittanyblake.github.io/NewYorkTimes-clone/)

### CODE REVIEW

**Review by fatymahmed — COMMENTED** [review]
## Hi @BrittanyBlake @Morrism1, 
Great job guys,👏👏 very nice description for your pull request 😀, there are just a few things that you need to change for it to be better and approved.

**Navigation bar**
- It has a "Hamburger **button**" on the leftmost side, kindly change it to a button.
- The NY Times logo should be a link.

**Main Content**
- Make sure your text is aligned with the image.
This is how it looks on the new york page.
screenshot
This is how it looks on yours.
screenshot

**Bottom part**
- For the _More in Space and Astronomy_ section, the images and texts below them are **links** to other articles.

**Footer**
- It should have a fixed position (it sticks to the bottom part of the screen) -  This is for the _Access more of the Times by creating a free account or logging in_ part.

**Implementation details**
**The Theory of Inflation**
- I noticed that you are using an image with the text included in the image for this part, however, you should use the image without the text (you can get it from the new york times page)then add the text by yourself.

**General**
- Check HTML markup with this validator https://validator.w3.org/#validate_by_upload as [mentioned in Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png), you have some errors, fix them.
Screenshot
- You could add some space between the main content and the aside section, this is how it looks.
screenshot

**[OPTIONAL]**
- You are encouraged to use the `<main>` tag for the dominant content of the `<body>` of a document.
-  For the use of absolute positioning for the editor's picks and subscribe sections, you can easily achieve the expected outcome with the use of the `<section>` and `<aside>` tags which help keep the document flow.

**Review by fatymahmed — CHANGES_REQUESTED** [review]
### Kindly make the suggested changes then request for another code review.
### Status: Changes Required
_Feel free to reach out to me on slack (Fatima) if you have any questions, I'd be happy to help._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,696 @@
...
+}
+
+.image_title {
+  height: 34em;
+  width: 59em;
+}
+
+.main .caption {
+  text-align: left;
+  margin-left: -170px;
```
Comment (fatymahmed) [inline style.css:75]: - You can make this be aligned with the image by changing the `margin-left` .

**Comment by BrittanyBlake** [comment]
## Positioning and Floating Elements: New York Times Clone
-Fixed navigation bar hamburger button
-Made NY Times logo a link
-Added links to images and text in "More in Space and Astronomy" section
-Added footer with fixed position
-Fixed HTML validator errors
-Added blank theory of inflation image
-Added text to the theory of inflation image


[Live Version](https://brittanyblake.github.io/NewYorkTimes-clone/)

**Review by nikasulo — APPROVED** [review]
# Really amazing work! @BrittanyBlake and @Morrism1 :handshake: 
This is really beautiful and I loved reviewing your project. Amazing job on the PR description as well, it was nice to read. Please merge your branch and keep moving forward. See you at the top! :rocket: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Pull request merged**